app/src/helpers/requests/uploadDocuments.ts

```diff
@@ -180,5 +188,29 @@ const uploadDocuments = async ({
         throw error;
     }
 };
+export const updateDocumentState = async ({
+    document,
+    uploadingState,
+    documentReference,
+    baseUrl,
+    baseHeaders,
+}: UpdateStateArgs) => {
+    const fileKey = documentReference.split('/')[3];
+    const updateUploadStateUrl = baseUrl + endpoints.UPLOAD_DOCUMENT_STATE;
+    const body = {
+        files: [
+            {
+                reference: fileKey,
+                type: document.docType,
+                fields: { Uploading: uploadingState },
+            },
+        ],
+    };
+    await axios.post(updateUploadStateUrl, body, {
```

Comment:
return this here, also needs a try catch, even if the catch is empty it prevents the page from breaking